Dockerfile

```diff
@@ -1,21 +1,66 @@
+ARG DOCKER_COMPOSE=docker/compose:1.24.1
+ARG DOCKER=docker:19.03.1
+ARG BUILDKIT=moby/buildkit:v0.6.1
+ARG BLDR=autonomy/bldr:946e61b-scratch
+
+FROM $DOCKER_COMPOSE as docker_compose
```

Comment:
Now that `hack/dev` is gone, I don't think we need this anymore. Not a big deal to me to keep it, just pointing that out.